docker-compose.yaml

```diff
@@ -0,0 +1,9 @@
+version: '3.7'
+services:
+  site:
+    container_name: docsy
+    command: jekyll serve --watch --drafts
+    image: jekyll/jekyll:latest
+    volumes: ["./:/srv/jekyll", "./vendor/bundle:/usr/local/bundle"]
```

Comment:
My intuition is that binding ./vendor/bundle to anywhere in /usr would be not a great idea, security wise. Could you clarify why this needs to be bound to `/usr/local/bundle` ? Or are you simply trying to put it somewhere so it's not found in the root of `/srv/jekyll` ?

Reply:
We are mounting from container to host not host to container. 

You can run without mounting bundle. On every restart bundle will be a clean build which is time consuming. This is optional. 

We are using the default folder provided by container, we are just mounting them to host to maintain persistency of bundles on every restart. 

To test it better remove the second volume and do ‘docker-compose down‘ and ‘docker-compose up‘

Second test add the volume and repeat the same.

Reply:
To understand more better read about jekyll/jekyll docker image official README.md

https://github.com/envygeeks/jekyll-docker/blob/master/README.md

Reply:
Ah, understood. And I just pulled the container to see what you meant - the bundle directory already exists in the container and you are saving build time (I wasn't familiar with jekyll/jekyll).

```bash
$ docker run -it --rm --entrypoint bash jekyll/jekyll:latest
bash-5.0# ls /usr/local/
bin      bundle   etc      include  lib      share
```
And I agree it would save time to mount and then store those files.